l10n/en/mozorg/home-m24.ftl

```diff
@@ -0,0 +1,96 @@
+m24-home-can-ai-be = Can AI be trustworthy?
+m24-home-in-a-world = In a world where AI innovation is driven by a select few, we’re risking monopolization of this technology. Applying open-source concepts to AI could change that.
+m24-home-watch-now = Watch now
+m24-home-mozilla-ventures = { -brand-name-mozilla } Ventures
+m24-home-got-an-earlystage = Got an early-stage startup? Pitch your company to { -brand-name-mozilla } Ventures and secure funding to drive positive change for the future of AI and the internet.
+m24-home-common-voice = { -brand-name-common-voice }
+m24-home-donate-your-voice-to-make = Donate your voice to make speech technology more inclusive and accessible for everyone.
+m24-home-donate-your-voice = Donate your voice
+
+## Media Springboard
+
+m24-home-type = Type
+m24-home-author = Author
+m24-home-topic = Topic
+m24-home-intro = Intro
+m24-home-tag-article = Article
+m24-home-tag-podcast = Podcast
+m24-home-tag-video = Video
+m24-home-topic-news = News
+m24-home-topic-ai = Artificial Intelligence
+m24-home-topic-ps = Privacy &amp; Security
+m24-home-headline-you-ai = Headline: You, AI and the internet — what’s really going on?
+m24-home-introducing-anonym = Introducing { -brand-name-anonym }: Raising the bar for privacy-preserving digital advertising.
+m24-home-keeping-genai-technologies = Keeping GenAI technologies secure is a shared responsibility.
+m24-home-from-hollywood-to = From Hollywood to hip hop, artists are negotiating AI consent.
+m24-home-mozilla-celebrates-groundbreaking = { -brand-name-mozilla } celebrates groundbreaking creators in new docuseries “{ -brand-name-firefox } Presents.”
+m24-home-big-techs-power = Big tech’s power over language means power over people. Bridget Todd talks to language community leaders paving the way for voice AI in their own languages and dialects.
+m24-home-why-does-it = Why does it so often feel like we’re part of a mass AI experiment? What is the responsible way to test new technologies? Bridget Todd explores what it means to live with unproven AI systems that impact millions of people as they roll out across public life.
+m24-home-kids-are-growing = Kids are growing up in a very online world. What’s a concerned parent to do?
+
+## Showcase
+
+m24-home-rise25-awards = Rise25 Awards
+m24-home-get-to-know = Get to know the leaders championing trustworthy, inclusive and transparent AI development as they set a new standard and inspire the next generation of innovators.
+m24-home-watch-the-award = Watch the award show
+
+## Issues
+
+m24-home-explore-issues-shaping = Explore issues shaping the future of the internet
+m24-home-privacy-not-included = *Privacy Not Included
+m24-home-search-our-reviews = Search our reviews to see which tech gadgets and apps are stealthily sneaking your data.
+m24-home-pni-search-now = Search now
+m24-home-irl-podcast = IRL Podcast
```

Comment:
**Question (non-blocking):** Should `IRL Podcast` also be a brand name?

Reply:
✅ I'll make it a brand term.